cypress/integration/app.spec.js

```diff
@@ -0,0 +1,120 @@
+//       expect(confirmText).eq("정말 삭제하시겠습니까?");
+//       return true;
+//     });
+// 
+//     cy.get("#product-control-table").children().eq(1).should("be.empty");
+//   });
+// 
+//   it("자판기에 같은 이름의 상품은 등록할 수 없다", () => {
+//     cy.get(".product-control-input").eq(0).type("사이다");
+//     cy.get(".product-control-input").eq(1).type(1000);
+//     cy.get(".product-control-input").eq(2).type(10);
+//     cy.get("#product-add-button").click();
+// 
+//     cy.on("window:alert", (alertText) => {
+//       expect(alertText).eq("중복된 상품명은 등록할 수 없습니다.");
+//     });
+//   });
+// 
+//   it(`자판기에 등록할 상품명은 ${PRODUCT.MAX_LENGTH}글자 까지 가능하다`, () => {
+//     cy.get(".product-control-input").eq(0).type("사이다를마시다가용트름");
+//     cy.get(".product-control-input").eq(1).type(1000);
+//     cy.get(".product-control-input").eq(2).type(10);
+//     cy.get("#product-add-button").click();
+// 
+//     cy.on("window:alert", (alertText) => {
+//       expect(alertText).eq(`상품명은 최대 ${PRODUCT.MAX_LENGTH}글자까지 입력 가능합니다.`);
+//     });
+//   });
+// 
+//     it(`상품 가격은 ${PRODUCT.MIN_PRICE}원부터, 최대 ${PRODUCT.MAX_PRICE}원까지 가능하다.`, () => {
+//     cy.get(".product-control-input").eq(0).type("콜라");
+//     cy.get(".product-control-input").eq(1).type(10001);
+//     cy.get(".product-control-input").eq(2).type(10);
+//     cy.get("#product-add-button").click();
+// 
+//     cy.on("window:alert", (alertText) => {
+//       expect(alertText).eq(`상품 가격은 ${PRODUCT.MIN_PRICE}원부터, 최대 ${PRODUCT.MAX_PRICE}원까지 가능합니다.`);
+//     });
+//   });
+// 
+//   it(`상품 가격은 ${PRODUCT.UNIT}원으로 나누어 떨어져야한다.`, () => {
+//     cy.get(".product-control-input").eq(0).type("콜라");
+//     cy.get(".product-control-input").eq(1).type(1001);
+//     cy.get(".product-control-input").eq(2).type(10);
+//     cy.get("#product-add-button").click();
+// 
+//     cy.on("window:alert", (alertText) => {
+//       expect(alertText).eq(`상품 가격은 ${PRODUCT.UNIT}원으로 나누어 떨어져야합니다.`);
+//     });
+//   });
+// 
+//     it(`제품당 수량은 최소 ${PRODUCT.MAX_QUANTITY}개부터 최대 ${PRODUCT.MIN_QUANTITY}개까지 가능하다.`, () => {
+//     cy.get(".product-control-input").eq(0).type("콜라");
+//     cy.get(".product-control-input").eq(1).type(1000);
+//     cy.get(".product-control-input").eq(2).type(11);
+//     cy.get("#product-add-button").click();
+// 
+//     cy.on("window:alert", (alertText) => {
+//       expect(alertText).eq(`제품당 수량은 최소 ${PRODUCT.MAX_QUANTITY}개부터 최대 ${PRODUCT.MIN_QUANTITY}개까지 가능합니다.`);
+//     });
+//   });
+// });
+
+describe("잔돈 관리 탭 테스트", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:9000/#product");
+    cy.get(".nav__button").eq(1).click();
+  });
+
+  it("잔돈을 충전하면 500, 100, 50, 10 원 중에서 반환된다", () => {
+    cy.get(".charge-control-input").type(40);
+    cy.get("#charge-add-button").click();
+    cy.get(".charge-coin-count").eq(3).should("have.text", "4개");
+  })
+
+  it(`최소 ${CHARGE.MIN_PRICE}원, 최대 ${CHARGE.MAX_PRICE}원까지 충전할 수 있다.`, () => {
+    cy.get(".charge-control-input").type(100001);
```

Comment:
```suggestion
    cy.get(".charge-control-input").type(MAX_PRICE + 1);
```
설정값에 맞게 바꿔주세요~ (다른 테스트도요!)